mentors/javascript/z.json

```
@@ -6,5 +6,13 @@
     "link_url": "https://github.com/zeckdude",
     "avatar_url": "https://avatars2.githubusercontent.com/u/947856",
     "bio": "I live, breathe, and dream JavaScript. I could be dreaming right now."
-  }
+  },
+  {
+  "github_username": "zimmah",
+  "name": "Zimri Leijen",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "Fullstack web developer using Javascript and React, cryptocurrency pioneer"
+}
```

```suggestion
    "github_username": "zimmah",
    "name": "Zimri Leijen",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "Fullstack web developer using Javascript and React, cryptocurrency pioneer"
  }
```

Just some minor formatting changes.